tests/unit/packet/PublishTest.php

```diff
@@ -5,285 +5,267 @@
  * Time: 16:35
  */
 
+use oliverlorenz\reactphpmqtt\packet\MessageHelper;
+use oliverlorenz\reactphpmqtt\packet\Publish;
+use oliverlorenz\reactphpmqtt\protocol\Version4;
+
+/**
+ * See http://docs.oasis-open.org/mqtt/mqtt/v3.1.1/os/mqtt-v3.1.1-os.html#_Toc398718038
```

Comment:
I just added the links to the spec here, as pretty much every method would need a link otherwise. I think those are the only two sections that are relevant. Hope that's ok.